### DISCUSSION TITLE
您好，我使用Taro开发小程序使用alova库，当页面经过小程序路由缓存后再次进入页面不会重新调用获取数据接口。

### DISCUSSION THREAD

**Discussion by Ryan-CW-Code** [alovajs/alova#269] [Q&A]
### 需要新特性解决什么问题？

我在第一个页面使用 useRequest 获取数据，经过小程序路由跳转后，再返回或者重新进入第一个页面，由于此**页面没有被卸载**不会再次触发获取数据。
可能导致数据刷新有很高的延时 / 必须用户手动刷新

### 你期望的新特性应该如何呈现？

我目前采用这样的方式，想知道作者是否有更好的方式来解决这个问题
![image](https://github.com/alovajs/alova/assets/75795462/93702578-e2fd-4da2-a78b-ed2606e846f5)

### 它对你有多重要？

可以让我更便于开发

**Comment by JOU-amjs** [comment]
问题请在Discussion栏目提问

我理解的是你想要让用户在再次进入这个页面时重新刷新数据，如果是这样的话推荐使用下`@alova/scene-react`的useAutoRequest，它可以做到这些。[文档在这](https://alova.js.org/tutorial/strategy/useAutoRequest)
